apps/docs/src/guide/scripts/script-with-configurable-constants.md

```diff
@@ -0,0 +1,21 @@
+# Script With Configurable
+
+In the same way as [Contracts](../contracts/configurable-constants.md) and [Predicates](../predicates/predicate-with-configurable-constants.md), Scripts also support configurable constants. This feature enables dynamic adjustment of certain values within your scripts.
+
+Configurable constants are fairly straightforward to add and set in your Scripts.
+
+Let's consider the following Script:
+
+<<< @/../../docs-snippets/contracts/sum-script/src/main.sw#script-with-configurable-contants-1{rust:line-numbers}
+
+In this Script, `AMOUNT` is a configurable constant with a default value of `10`. The main function returns the sum of the `inputted_amount` and the configurable constant `AMOUNT`.
```

Comment:
@arboleya I would opt for `ClassName` when referencing the code specifically. Here though it seems like we are referencing the script as a process/concept, and therefore not in code so double ticks does not seem appropriate to me.

Reply:
@arboleya, you've got a point there. Since this sentence is referring to a Sway script and not to a Script class, maybe we should just stick with "script", just like the Sway docs. I've noticed use scripts, predicates, and contracts straight up.